docs/tutorial/conversion.rst

```diff
@@ -0,0 +1,292 @@
+            },
+        },
+        'required': ['agent', 'election'],
+    }
+
+Use this schema and adapt the ``__str__()`` and ``summary()`` methods.
+
+Since the schema is distributed in a separate python package, it must be added
+to the ``election`` app's dependencies in ``requirements.txt``.
+
+Wrapper function
+~~~~~~~~~~~~~~~~
+Now you can import this class in ``fedora_elections/fedmsgshim.py`` and use it
+to encapsulate the messages. The wrapper could look like::
+
+    import logging
+
+    from fedora_elections_message_schemas.schema import Message
+    from fedora_messaging.api import publish as fm_publish
+    from fedora_messaging.exceptions import PublishReturned, ConnectionException
+
+    LOGGER = logging.getLogger(__name__)
+
+    def publish(topic, msg):
+        try:
+            fm_publish(Message(
+                topic="fedora.elections." + topic,
+                body=msg,
+            ))
+        except PublishReturned as e:
+            LOGGER.warning(
+                "Fedora Messaging broker rejected message %s: %s",
+                msg.id, e
+            )
+        except ConnectionException as e:
+            LOGGER.warning("Error sending the message %s: %s", msg.id, e)
+
+
+With this you'll get a couple of nice features over the previous state of
+things:
+
+- the message format is validated, so it's your responsability to update the
+  schema when you decide to change the format, and not the receiver's
+  responsability to handle any database schema changes you may make that may
+  bleed into the message dictionary. And you'll know during development if you
+  break compatibility.
+- you may handle messaging errors in anyway you deem relevant. Here we're just
+  logging them but you could choose to re-send the messages, store them for
+  further analysis, etc.
+- when there are no exceptions, you know that the message has reached the
+  broker and has been distributed.
+
+Testing
+~~~~~~~
+Let's start the election app and make sure messages are properly sent on the
+bus. First, we'll create a virtualenv, and install election and
+fedora-messaging with the following commands::
+
+    virtualenv venv
+    source ./venv/bin/activate
+    pushd elections-message-schemas
+    python setup.py develop
+    popd
+    pip install -r requirements.txt
+    python setup.py develop
+
+Make sure the Fedora Messaging configuration file is correct in
+``/etc/fedora-messaging/config.toml``. We will add a queue binding to route
+messages with the ``fedora.elections`` topic to the ``tutorial`` queue. Add
+this entry in the ``bindings`` list::
+
+    [[bindings]]
+    queue = "tutorial"
+    exchange = "amq.topic"
+    routing_keys = ["fedora.elections.#"]
+
+You could also add ``"fedora.elections.#"`` to the ``"routing_keys"`` value in
+the existing entry.
+
+Now make sure that RabbitMQ is still running, and run the ``consume.py`` script we
+used before. Make sure it is not systematically raising exceptions in the
```

Comment:
It can be helpful to link to the usage.rst document here where the consume.py script is first mentioned, just in case the reader forgot where they saw it